src/main/kotlin/com/autonomousapps/internal/utils/collections.kt

```diff
@@ -25,6 +25,10 @@ internal inline fun <T, R> Iterable<T>.mapToSet(transform: (T) -> R): HashSet<R>
   return mapTo(HashSet(collectionSizeOrDefault(10)), transform)
 }
 
+internal inline fun <T, R> Iterable<T>.mapToLinkedHashSet(transform: (T) -> R): HashSet<R> {
```

Comment:
Do you think we should change the pre-existing `maptToSet()` function to use a `LinkedHashSet` instead of adding a new method? Or, why do you prefer a linked hash set specifically here?

Reply:
I'd be down for that, as that would match kotlin's conventional behavior as well (`toSet()` guarantees preserved order)

Reply:
As for why LinkedHashSet - https://publicobject.com/2016/02/08/linkedhashmap-is-always-better-than-hashmap/

Reply:
a3c0509